webpack.config.js

```diff
@@ -138,6 +138,11 @@ var config = {
     ]
   },
   plugins: [
+    new LodashModuleReplacementPlugin({
+      collections: true,
+      currying: true,
+      flattening: true
```

Comment:
We talked about having a comment about mentioning `currying` and `flattening` are enabled to support react-mentions. Can we add that?